releases/22.10.0/22100 feature list.md

```diff
@@ -27,9 +29,11 @@ The general theme of this release is "TBD".
 * The Project Manager now displays the engine name and version number each project is registered with. The Project Manager now displays the current engine version in the title bar.
 * Script Canvas Architecture update that provides a framework to embed ScriptCanvas functionality in places other than Entity / Component system via the ScriptCanvas Component. The new architecture introduces a set of classes that handle a small part of the ScriptCanvas runtime. The highest level class is the ScriptCanvas::Executor which is now used by the ScriptCanvas::RuntimeComponent. The ScriptCanvas::Executor is also used by the new ScriptCanvasEditor::Interpreter, which is a class that allows developers to embed user access to ScriptCanvas functionality any where in the editor.
 * Writing C++ Script Canvas nodes is now a lot easier! We have deprecated node generics (a set of C++ macros) in favor of using AzAutoGen to produce libraries of functions or standalone nodes. We have consolidated the autogen semantics for Script Canvas grammar nodes and nodeables. We removed the concept of Script Canvas node libraries, anytime you write a node it gets registered automatically. It is no longer necessary to manually reflect or register Script Canvas nodes.
+* Prefab Edit Mode: Now when you double-click on a prefab you now go into prefab edit mode.
 * The O3DE editor viewport now comes with a new effect to help show both selected entities (using an outline effect) as well as helping you know what Prefab you're editing by applying a screen effect to everything not in focus. ([Editor Mode Visual Feedback Improvements (Prefab Edit Mode) #3458](https://github.com/o3de/o3de/issues/3458)).
```

Comment:
update: As we're not calling out the Prefab Edit Mode functionality explicitly above, we can update this line to not include `as well as helping you know what Prefab you're editing by applying a screen effect to everything not in focus`. So this just becomes

```
The O3DE editor viewport now comes with a new feature to help show selected entities with an outline effect.
```

Reply:
thanks. Updated with this proposed text.